### ISSUE TITLE
Translation: please explain an original sentence

### ISSUE THREAD

**Issue by severinmenard** [fieldpapers/fieldpapers#148]
Unstructured Text - Field Papers allows you to write notes about each feature when it’s added to the map. 

What "when it's added" is related to? Editing the notes in an OSM editor?

**Comment by mojodna** [comment]
This refers to the [barely used] "notes" functionality that's been removed, which was a tool for drawing points and polygons in a browser, using the snapshot as a background.

Was this in Transifex or `app/views/home/advanced.fr.html.erb`?

If the template, I removed a few of the bullets a couple hours ago after realizing that they referred to functionality that doesn't exist (in one case, not in the old version of FP either).

If Transifex, I'll track it down and update the translations.

**Comment by severinmenard** [comment]
yes it is on this file I am translating. Was about to ask what "you'd add a note for each feature on Field Papers, noting the uniqueID." means within:


Structured Data - Need to store structured form data about each map feature? Field Papers provides a large text field and you have two options: (a) use key:value pairs or (b) enter a uniqueID for a data join in a desktop GIS app with a worksheet of structured form data. An example of a key:value pair entered into the notes widget: type:park, name:Golden Gate, about:Large urban park with museums and buffalo. You can use another program outside of Field Papers to split those into separate columns. An example of a uniqueID is using a series of sequential numbers (1, 2, 3) entered into the notes widget like: 1. You'd fill out a separate stack of structured data "forms" about each feature in the field, marking the sequential uniqueID onto the map, onto it's form, filling out the form. Back at the computer, you'd add a note for each feature on Field Papers, noting the uniqueID. You'd transcribe the forms into a worksheet using OpenOffice or Excel noting both the uniqueID and the structured data about that feature. Download the GeoJSON or SHP from Field Papers, export your worksheet data, and use a program like QGIS or ArcGIS to "join" the two together. 


So it does not exist and the text will be removed, right?

**Comment by mojodna** [comment]
Correct.  https://github.com/fieldpapers/fp-web/commit/6f7302d5b577aab6878a0ce9c1ae1ad836b0776a has the relevant changes.

The forms functionality was implemented a few years back, but it doesn't look like it ever got enabled.
